README.md

```diff
@@ -458,6 +458,109 @@ The `ModuleProtocol` class used for resolving, reading and loading modules. See
 
 Unloads the module.
 
+### CommonJS Import Methods
+
+#### `require(specifier, options = {})`
+
+Used to import modules, JSON and local files. Relative paths (e.g. `./`, `./foo`, `./bar/baz`, `../foo`) will be resolved against the directory named by `__dirname` (if defined) or the current working directory. The relative paths of POSIX style are resolved in an OS independent fashion, meaning that the examples above will work on Windows in the same way they would on Unix systems.
```

Comment:
```suggestion
Used to import JavaScript or JSON modules and local files. Relative paths such as `./`, `./foo`, `./bar/baz`, and `../foo` will be resolved against the directory named by `__dirname`. POSIX style paths are resolved in an OS independent fashion, meaning that the examples above will work on Windows in the same way they would on POSIX systems.
```

Reply:
@kasperisager a nitpick regarding:
> in the same way they would on _POSIX_ systems.

Since this sentence is contrasting operating systems (Windows vs Unix), I think it makes sense to leave it as 'Unix' instead of changing to 'POSIX'. The rest of the suggestions makes sense to me.

Reply:
Unix is a very specific brand of operating systems and POSIX is a set of open standards that specify APIs for Unix-like systems. We should only ever reference the POSIX standards, not the Unix brand.

Reply:
It's also why the `node:path`/`bare-path` API is `path.posix.sep` and `path.win32.sep`

Reply:
Ah, you want the emphasize that it works with either of the OS interfaces instead of giving examples of how it can be run on different OSs. Sounds good to me.